### ISSUE TITLE
Tools ecosystem processes

### ISSUE THREAD

**Issue by hmenager** [bio-tools/content#455]
This table is meant to document the various processes (workflows) taking place in this repository:

| Source name       | Import           | Validation  | Cross-link  | Report | Publish | 
| ----------------------- | :-----------------: | :-------------: | :--------------: | :---------: | :----------: |
| bio.tools               | [biotools-import.yaml][1] | [biotools-testrunregistry.yml][2] | [doi-load.yaml][3] | [biotools-pullrequest-analyzer.yml][4] | [bioschemas-build.yml][5] [bioschemas-build-dump.yml][6] |
| bioconda              ||||||
| biocontainers       ||||||
| OpenEBench       ||||||
| OpenEBench metrics ||||||
| Debian Med               | [debianmed-import.yml][7] | [debian-yaml-validator.yaml][8] |  [doi-load.yaml][3] |||
| BIII | [biii-import.yaml][9] |||| [biii-import.yaml][9] |
| Global |||| [report.yaml][10] ||

[1]: https://github.com/bio-tools/content/blob/master/.github/workflows/biotools-import.yml
[2]: https://github.com/bio-tools/content/blob/master/.github/workflows/biotools-testrunregistry.yml
[3]: https://github.com/bio-tools/content/blob/master/.github/workflows/doi-load.yaml
[4]: https://github.com/bio-tools/content/blob/master/.github/workflows/biotools-pullrequest-analyzer.yml
[5]: https://github.com/bio-tools/content/blob/master/.github/workflows/bioschemas-build.yml
[6]: https://github.com/bio-tools/content/blob/master/.github/workflows/bioschemas-build-dump.yml
[7]: https://github.com/bio-tools/content/blob/master/.github/workflows/debianmed-import.yml
[8]: https://github.com/bio-tools/content/blob/master/.github/workflows/debian-yaml-validator.yaml
[9]: https://github.com/bio-tools/content/blob/master/.github/workflows/biii-import.yaml
[10]: https://github.com/bio-tools/content/blob/master/.github/workflows/report.yaml

**Comment by hmenager** [comment]
@OlegZharkov @bgruening can you provide the bioconda + biocontainers information?

**Comment by hmenager** [comment]
@redmitry from what I understand openebench-git-populous only sends the metrics data. Do you have another script to send the openebench data themselves?

**Comment by redmitry** [comment]
> @redmitry from what I understand openebench-git-populous only sends the metrics data. Do you have another script to send the openebench data themselves?
No. The reason is that OpenEBench tools collected from bio.tools, bioconda and some time ago galaxy. That would be a duplication (?).  
I have a code to import bio.tool -> git:  
https://gitlab.bsc.es/inb/elixir/tools-platform/biotools-git-populous

There is no problem to push OpenEBench tools to the github, but its update supposed to be done on the over direction (from conda/biotools files).

Cheers,

D.

**Comment by hmenager** [comment]
Thanks for the clarification @redmitry. Indeed, if you have no other source for metadata than bio.tools and bioconda, then there is probably no need to document what is probably a "one-time" import.

**Comment by rioualen** [comment]
This one might need some refreshing?

**Comment by rioualen** [comment]
This issue pre-dates the creation of the repository `utils`, and thus rather outdated. 

A recent, non-exhaustive alternative can be seen in #1609
